driver/kubernetes/context/fixtures/k3s-kubeconfig

```diff
@@ -9,8 +9,8 @@ contexts:
     cluster: test-cluster
     user: test-user
     namespace: zoinx
-  name: test
-current-context: test
+  name: k3s
+current-context: k3s
```

Comment:
?

Reply:
I changed context to test if precedence matches expectations in the new test: https://github.com/docker/buildx/pull/2497/files#diff-bf08b5ea9b0c961aad5fc56bdec629b3cd9d42a21121839e0d5445a6effb4d92R22-R35